src/cfg.c

```diff
@@ -190,6 +191,39 @@ bool cfg_set_border_width_monocle(const char *s) {
 	}
 }
 
+bool cfg_set_layout_format(const char *s) {
```

Comment:
I don't quite understand the failure cases of this function; I might be missing the point, but could it just be "garbage in, garbage out"?

Can you please add tests to `tst-args_cli.c` to cover these cases?